guardian/shortcuts.py

```diff
@@ -159,7 +170,7 @@ def get_perms_for_model(cls):
     return Permission.objects.filter(content_type=ctype)
 
 def get_users_with_perms(obj, attach_perms=False, with_superusers=False,
-        with_group_users=True):
+        with_group_users=True, direct_perms_only=False):
```

Comment:
In which way it is a backwards incompatible change? It didn't break any tests.

You mean, because now `attach_perms` set to `True` return really what you asked for and not everything as before? I see this as a bugfix, but yes, it should be mentioned in the release notes.